modules/messages/src/lib.rs

```diff
@@ -398,7 +398,7 @@ pub mod pallet {
 
 			Self::deposit_event(Event::MessagesReceived(messages_received_status));
 
-			Ok(PostDispatchInfo { actual_weight: Some(actual_weight), pays_fee: Pays::No })
+			Ok(PostDispatchInfo { actual_weight: Some(actual_weight), pays_fee: Pays::Yes })
```

Comment:
@svyatonik 
just thinking, is it worth to have some tests for extrinsics what is free and what not? on pallet level or runtime level?

Reply:
Yeah,, let's add one here. Actually - messages transactions were never intended to be free, so it's just a typo. But in anyway - let's do this